### ISSUE TITLE
How to get raw response body / responseData?

### ISSUE THREAD

**Issue by CrabDude** [hallas/agent#18]
Agent seems to expect JSON response by default, returning an error otherwise.

```swift
Agent.get("http://google.com").end {
  response, data, error in
  println(error)
}

/*
Error Domain=NSCocoaErrorDomain Code=3840 "The operation couldn’t be completed. (Cocoa error 3840.)" (JSON text did not start with array or object and option to allow fragments not set.) UserInfo=0x7fa9f1469210 {NSDebugDescription=JSON text did not start with array or object and option to allow fragments not set.}
*/
```

How can I get either the body as a string or as responseData to coerce to a string?

**Comment by hallas** [comment]
Will be implementing functions that coerce responses into the data format you need, starting with the raw data and JSON. From there you can extend it as you need. Please keep track in:

https://github.com/hallas/agent/pull/26

**Comment by hallas** [comment]
Merged in #26 with out this but will start a new feature branch for this now.

**Comment by hallas** [comment]
Should be fixed in #29
